src/uproot/models/RNTuple.py

```diff
@@ -457,17 +456,56 @@ def arrays(
             [c.num_entries for c in clusters[start_cluster_idx:stop_cluster_idx]]
         )
 
+        self._alias_columns_dict = {
+            el.field_id: el.physical_id
+            for i, el in enumerate(self.header.alias_columns)
+        }
+        self._column_recorrds_dict = {}
```

Comment:
Very minor comment: recorrds -> records?

Reply:
You're right, thanks! See #1088.